netbox/circuits/forms.py

```diff
@@ -144,7 +144,7 @@ class Meta:
         ]
 
 
-class CircuitTypeCSVForm(forms.ModelForm):
+class CircuitTypeCSVForm(CustomFieldForm):
```

Comment:
Only models which support the assignment of custom fields (those identified in `extras.constants.CUSTOMFIELD_MODELS`) would need to have their import forms changed.